token-group/js/src/state/tokenGroup.ts

```diff
@@ -0,0 +1,72 @@
+import { PublicKey } from '@solana/web3.js';
+import { getBytesCodec, getStructCodec } from '@solana/codecs-data-structures';
+
+const tokenGroupCodec = getStructCodec([
+    ['updateAuthority', getBytesCodec({ size: 32 })],
+    ['mint', getBytesCodec({ size: 32 })],
+    ['size', getBytesCodec({ size: 4 })],
+    ['maxSize', getBytesCodec({ size: 4 })],
+]);
+
+export class PodU32 {
+    #value: number;
+    constructor(value: number) {
+        this.#value = value;
+    }
+    toBuffer(): Buffer {
+        const buffer = Buffer.alloc(4);
+        buffer.writeUInt32LE(this.#value);
+        return buffer;
+    }
+}
```

Comment:
I think we can cut this, so that the JavaScript user is always interacting with `number` in the context of this interface. Then when we pack/unpack, we can do the proper byte allocation for `u32`. What do you think?

Reply:
souds good